framework/tlvf/yaml/tlvf/ieee_1905_1/tlvWscM2.yaml

```
@@ -0,0 +1,13 @@
+#
+---
+_include: tlvf/WSC/sM2.h
+_namespace: ieee1905_1
+_is_tlv_class : True
+
+tlvWscM2:
+  _type: class
+  type:
+    _type: eTlvType
+    _value_const: TLV_WSC
+  length: uint16_t
+  M1Frame: WSC::sM2
```

I didn't notice before:
```suggestion
  M2Frame: WSC::sM2
```